config/crd/bases/etcd.aenix.io_etcdclusters.yaml

```diff
@@ -39,6 +39,86 @@ spec:
           spec:
             description: EtcdClusterSpec defines the desired state of EtcdCluster
             properties:
+              podDisruptionBudget:
```

Comment:
Could you explain please?

Reply:
I mean that sample CR should include PDB settings. It will be an example usage and anybody can install easily install it

Reply:
Basic PDB configuration is `spec.podDisruptionBudget: {}`, it will generate PDB resource with `minAvailable` configuration according to the number of replicas in the cluster. I don't think more options in sample is necessary. We can add them later to docs

Reply:
It is fine. This comment was written before sample CR was changed and pdb creation was implemented.